app/views/admin/users/index.html.haml

```diff
@@ -1,10 +1,14 @@
 .row
   .col-md-12
     .page-header
-      %h2
+      %h1
         Users
         - if @users
           = "(#{@users.length})"
+
+        - if can? :create, User && ENV['OSEM_ICHAIN_ENABLED'] != 'true'
```

Comment:
@differentreality commits squashed, I've also adjusted the code to get rid of the flash vs flash.now problem that we discussed recently (ref #1429)